.github/workflows/prod-build.yml

```
@@ -0,0 +1,71 @@
+name: 'Production Release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        required: true
+        type: choice
+        description: Which package should be published?
+        options:
+          - vue
+          - angular
+          - react
+      version:
+        required: true
+        type: choice
+        description: Which version should be published?
+        options:
+          - patch
+          - minor
+          - major
+          - prepatch
+          - preminor
+          - premajor
+          - prerelease
+      tag:
+        required: true
+        type: choice
+        description: Which npm tag should this be published to?
+        options:
+          - latest
+          - next
+      preid:
+        type: choice
+        description: Which prerelease identifier should be used? This is only needed when version is "prepatch", "preminor", "premajor", or "prerelease".
+        options:
+          - ''
+          - alpha
+          - beta
+          - rc
+          - next
+
+jobs:
+  dev-build:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.dev-build.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
```

Can we use the latest version of this action? ATM, `actions/checkout` is on v3. (3.5.2, specifically):

```suggestion
      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```

---

Punting til next PR